### PR TITLE
[CI] Rolling stable release 821

### PR DESCRIPTION
**Pull request by pbchekin**
Build a new runner image with packages from the recent rolling stable release 821.
Newly skipped tests are tracked in #797, #800.
Pass rate: 92.76% -> 92.6%

### CODE REVIEW

File: python/test/unit/language/test_subprocess.py [hunk 1]
```diff
@@ -34,6 +34,13 @@ def test_print(func_type: str, data_type: str):
     outs, _ = proc.communicate()
     outs = [line for line in outs.decode("UTF-8").split("\n") if line]
 
+    if (func_type, data_type) == ('print', 'int32'):
```
Comment (whitneywhtsang) [inline python/test/unit/language/test_subprocess.py:37]: These tests pass for me when only upgrading libigc1, do you know which component cause the failures?

File: .github/workflows/build-test.yml [hunk 2]
```diff
@@ -165,7 +165,7 @@ jobs:
           python3 05-layer-norm.py
           python3 06-fused-attention.py
           python3 07-extern-functions.py
-          python3 08-grouped-gemm.py
+          python3 08-grouped-gemm.py || true
```
Comment (whitneywhtsang) [inline .github/workflows/build-test.yml:168]: Please add a FIXME comment here

File: python/test/unit/language/test_subprocess.py (hunk 1 above)
Comment (pbchekin) [reply python/test/unit/language/test_subprocess.py:37]: No, we need a closer look. Potentially something simple.

Comment (whitneywhtsang) [reply python/test/unit/language/test_subprocess.py:37]: Can you please split the issue for these print failures from the other failures, as all the other failures are due to IGC.

File: .github/workflows/build-test.yml (hunk 2 above)
Comment (pbchekin) [reply .github/workflows/build-test.yml:169]: Done

File: python/test/unit/language/test_subprocess.py (hunk 1 above)
Comment (pbchekin) [reply python/test/unit/language/test_subprocess.py:37]: Moved to issue #800.

Comment (whitneywhtsang) [reply python/test/unit/language/test_subprocess.py:37]: Thanks Pavel

**Review by whitneywhtsang — APPROVED** [review]

**Review by leshikus — COMMENTED** [review]
LGTM

**Pull request merged**